editor/import/editor_scene_importer_gltf.h

```diff
@@ -218,6 +222,23 @@ class EditorSceneImporterGLTF : public EditorSceneImporter {
 		GLTFCamera() {}
 	};
 
+	struct GLTFLight {
+		Color color;
+		float intensity;
+		String type;
+		float range;
+		float inner_cone_angle;
+		float outer_cone_angle;
+		GLTFLight() {
+			color = Color(1.0f, 1.0f, 1.0f);
+			intensity = 1.0f;
+			type = "";
+			range = Math_INF;
+			inner_cone_angle = 0.0f;
+			outer_cone_angle = Math_PI / 4.0;
+		}
```

Comment:
Initialize values directly on declaration.

Reply:
Nice catch, fixed. By the way, is there any way for CI to catch this?

Reply:
Not that I know of :/
`clang-tidy` can fix it but it's very unwieldy and produces tons of false positives breaking the codebase (and can't be disabled for `thirdparty/`).